src/components/DataMessenger/DataMessenger.js

```diff
@@ -201,7 +199,9 @@ export default class DataMessenger extends React.Component {
         // Update the component so that the message sizes autoadjust again
         this.forceUpdate()
       }
-
+      if (this.state.isSizeMaximum !== prevState.isSizeMaximum) {
+        this.forceUpdate()
+      }
```

Comment:
Only when user drag the DM manually **_from the Full Screen mode_**, this will be triggered. This is intended to change the FullScreen icon to Exit FullScreen icon.